src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -191,10 +197,32 @@ public void selectDeck(Index deckIndex) {
     @Override
     public void unselectDeck() {
         this.selectedDeck = null;
+        updateFilteredCardList(PREDICATE_SHOW_ALL_CARDS);
```

Comment:
Just to clarify, `updateFilteredCardList` is the list of cards filtered by the `selectedDeck` right

Reply:
Yes.